apps/server/src/modules/learnroom/controller/rooms.controller.ts

```diff
@@ -58,4 +60,14 @@ export class RoomsController {
 		const dto = CopyMapper.mapToResponse(copyStatus);
 		return dto;
 	}
+
+	@Post('lessons/:lessonid/copy')
```

Comment:
yes, that was intended, also because lessons are available be only from the room
second option - is what we have now in legacy, but we don't need the roomId